doc/neps/nep-0028-website-redesign.rst

```diff
@@ -0,0 +1,302 @@
+
+The following should be considered when choosing a static site generator:
+
+1. *How widely used is it?* This is important when looking for help maintaining
+   or improving the site. More popular frameworks are usually also better
+   maintained, so less chance of bugs or obsolescence.
+2. *Ease of deployment.* Most generators meet this criterion, however things
+   like built-in support for GitHub Pages helps.
+3. *Preferences of who implements the new site.* Everyone has their own
+   preferences. And it's a significant amount of work to build a new site.
+   So we should take the opinion of those doing the work into account.
+
+Traffic
+```````
+
+The current site receives on the order of 500,000 unique visitors per month.
+With a redesigned site and relevant content, there is potential for visitor
+counts to reach 5-6 million -- a similar level as
+`scipy.org <http://scipy.org>`_ or `matplotlib.org <http://matplotlib.org>`_ --
+or more.
+
+Possible options for static site generators
+```````````````````````````````````````````
+
+1. *Jekyll.* This is a well maintained option with 855 Github contributors,
+   with contributions within the last month. Jekyll is written in Ruby, and
+   has a simple CLI interface. Jekyll also has a large directory of
+   `themes <https://jekyllthemes.io>`__, although a majority cost money.
+   There are several themes (`serif <https://jekyllthemes.io/theme/serif>`_,
+   `uBuild <https://jekyllthemes.io/theme/ubuild-jekyll-theme>`_,
+   `Just The Docs <https://jekyllthemes.io/theme/just-the-docs>`_) that are
+   appropriate and free. Most themes are likely responsive for mobile, and
+   that should be a requirement. Jekyll uses a combination of liquid templating
+   and YAML to render HTML, and content is written in Markdown. i18n
+   functionality is not native to Jekyll, but can be added easily.
+2. *Hugo.* This is another well maintained option with 554 contributors, with
+   contributions within the last month. Hugo is written in Go, and similar to
+   Jekyll, has a simple to use CLI interface to generate static sites. Again,
+   similar to Jekyll, Hugo has a large directory of
+   `themes <https://themes.gohugo.io>`_. These themes appear to be free,
+   unlike some of Jekyll's themes.
+   (`Sample landing page theme <https://themes.gohugo.io/hugo-hero-theme>`_,
+   `docs theme <https://themes.gohugo.io/hugo-whisper-theme>`_). Hugo uses Jade
+   as its templating language, and content is also written in Markdown. i18n
+   functionality is native to Hugo.
+3. *Docusaurus.* Docusaurus is a responsive static site generator made by Facebook.
+   Unlike the previous options, Docusaurus doesn't come with themes, and thus we
+   would not want to use this for our landing page. This is an excellent docs
+   option written in React. Docusaurus natively has support for i18n (via
+   Crowdin_, document versioning, and document search.
+
+Both Jekyll and Hugo are excellent options that should be supported into the
```

Comment:
isn't sphinx a viable 4th option, based on the pandas experience?

Reply:
Based on a decade of experience, I think no. There are literally zero sites I can think of that pulled this off with Sphinx. And https://pandas.pydata.org/ does not look good imho.

Reply:
since there's a longer conversation on this topic, I'll resolve this one.